src/originalBudgetTrackingApp/transactions/dates.ts

```diff
@@ -0,0 +1,5 @@
+export const formatDate = (date: Date) => `${date.getDate()}/${date.getMonth()}/${date.getFullYear()} ${date.getHours()}:${date.getMinutes()}`;
```

Comment:
Strange, maybe it was used in `.js` file. I think we will use it since I want to reduce the usage of the deprecated `moment` library, so sometime we will add this again, this is nothing.